spotify_project/manage.py

```diff
@@ -1,12 +1,25 @@
 #!/usr/bin/env python
-"""Django's command-line utility for administrative tasks."""
```

Comment:
I don't think we should be changing anything in here. I'll show you how when you get here later.